api/src/main/java/eu/europeana/fulltext/api/service/FTService.java

```diff
@@ -77,20 +78,21 @@ public AnnoPage fetchAnnoPage(String datasetId, String localId, String pageId, L
     }
 
     /**
-     * Retrieve the AnnoPage with the provided datasetId, localId and imageId. If the annotationType is specified the
-     * returned AnnoPage will only contain annotations of that type. If annotationType is null or empty then all
+     * Retrieve AnnoPages with the provided datasetId, localId and imageIds. If the annotationType is specified the
+     * returned AnnoPages will only contain annotations of that type. If annotationType is null or empty then all
      * annotations of that type will be returned
      * @param datasetId ID of the dataset
      * @param localId   ID of the parent of the Annopage object
-     * @param imageId   ID of the image
+     * @param imageIds   IDs of the images
      * @param annotationType type of annotations that should be retrieved, if null or empty all annotations of that
      *                        annopage will be retrieved
      * @return null if it doesn't exists, otherwise the retrieve annopage
      */
```

Comment:
I think we also need to update this and mention that it's the responsibility of the caller to iterate over the AnnoPages